modules/gradle-clojure-tools/build.gradle

```diff
@@ -7,6 +7,8 @@ plugins {
 dependencies {
   // gradle
   compileOnly gradleApi()
+  compileOnly 'org.clojure:clojure:1.8.0'
+  compileOnly 'org.projectodd.shimdandy:shimdandy-api:1.2.0'
```

Comment:
I didn't know about shimdandy before... it's pretty slick.  And a very nice addition to the plugin.